### PR TITLE
Added output for 0 results in !nano

### PR DESCRIPTION
**Pull request by Teeko1993**

### CODE REVIEW

File: modules/standard/nano/nano_controller.py [hunk 1]
```diff
@@ -34,7 +34,9 @@ def nano_cmd(self, request, search, named_params):
 
         blob = ""
 
-        if count == 1:
+        if count == 0:
+            return "No results found for <highlight>%s</highlight> " % search
+        elif count == 1:
             row = data[0]
             return self.format_single_nano(row)
         if count > page_size:
```
Comment (bigwheels16) [inline modules/standard/nano/nano_controller.py:42]: use `elif` here as well

File: modules/standard/nano/nano_controller.py [hunk 2]
```diff
@@ -34,7 +34,9 @@ def nano_cmd(self, request, search, named_params):
 
         blob = ""
 
-        if count == 1:
+        if count == 0:
+            return "No results found for <highlight>%s</highlight> " % search
```
Comment (bigwheels16) [inline modules/standard/nano/nano_controller.py:38]: Remove the space at the end, and add a period

**Review by bigwheels16 — CHANGES_REQUESTED** [review]

File: modules/standard/nano/nano_controller.py (hunk 2 above)
Comment (Teeko1993) [reply modules/standard/nano/nano_controller.py]: Updated, and matched the wording from !items output.

**Comment by bigwheels16** [comment]
looks good

**Pull request merged**